test/upgrade/dynamic/service.go

```diff
@@ -0,0 +1,265 @@
+package dynamic
```

Comment:
<!--copyright-bot-->

Missing copyright header, expected comment of the form:

    /*
    Copyright 2019 Something something authors.
    ...
    */